srcpkgs/gotify-server/template

```diff
@@ -0,0 +1,46 @@
+# Template file for 'gotify-server'
+pkgname=gotify-server
+version=2.1.3
+revision=1
+_git_commit=89fdb0b9a56690b7d1111783aa4fa73d61831e74
+wrksrc="server-$version"
+build_style=go
+go_import_path="github.com/gotify/server/v2"
+go_ldflags="-w -s -extldflags=-fuse-ld=bfd \
+ -X main.Version=${version} \
+ -X main.BuildDate=$(date "+%F-%T") \
+ -X main.Commit=${_git_commit} \
```

Comment:
If this isn't required, I would leave it out, this just complicates updates and I think it's not useful.